index.html

```diff
@@ -2,8 +2,7 @@
 layout: default
 class: home
 hide_breadcrumbs: true
-title: Choosing an OSS license doesn’t need to be scary
-description: A site to provide non-judgmental guidance on choosing a license for your open source project
+title: Choosing an open source license doesn’t need to be scary
```

Comment:
Changing the title in the YAML front matter updates the H1. Bumped the font size down to 60 from 63 and things seem better for both humans and robots IMHO:

![screen shot 2015-10-24 at 1 13 14 pm](https://cloud.githubusercontent.com/assets/282759/10711687/038ac5a4-7a51-11e5-9980-5cb4e55d2b52.png)